scalding-core/src/main/scala/com/twitter/scalding/Mode.scala

```diff
@@ -118,7 +127,9 @@ trait HadoopMode extends Mode {
         asMap - jarKey
       case None => asMap
     }
-    new HadoopFlowConnector(finalMap.asJava)
+    val clazz = Class.forName(jobConf.get("cascading.flow.connector.class", "cascading.flow.hadoop.HadoopFlowConnector"))
```

Comment:
Yes, it'd make sense to wrap classloader exceptions here and at least steer the user to the wiki, as already done on other exceptions. And on the wiki, a longer explanation that the user should provide the appropriate cascading fabric jar.

Will update that on Monday! Thanks for the suggestion.